internal/testdata/expected/javascript/ir/statements.js.out

```diff
@@ -0,0 +1,63 @@
+file statements.js:
+  func  ExportStatement ()
+  func  ForInStatement  ()
+  func  ForStatement    ()
+  func  IfStatement     (a, b)
+  func  SwitchStatement ()
+  func  TryStatement    ()
+  func  WhileStatement  ()
+
+
+
+============================
+
+# Name: ExportStatement
+# File: statements.js
+# Location: statements.js:99:0
+func ExportStatement():
+0:                                                                         entry
+
+# Name: ForInStatement
+# File: statements.js
+# Location: statements.js:91:0
+# Locals:
+#   0:	values
+func ForInStatement():
+0:                                                                         entry
+	values = nil
+
+# Name: ForStatement
+# File: statements.js
+# Location: statements.js:85:0
+func ForStatement():
+0:                                                                         entry
+
+# Name: IfStatement
+# File: statements.js
+# Location: statements.js:17:0
+func IfStatement(a, b):
+0:                                                                         entry
+	return a
+1:                                                                   unreachable
+
+# Name: SwitchStatement
+# File: statements.js
+# Location: statements.js:67:0
+# Locals:
+#   0:	fruits
+func SwitchStatement():
+0:                                                                         entry
+	fruits = "Oranges"
+
+# Name: TryStatement
+# File: statements.js
+# Location: statements.js:30:0
+func TryStatement():
+0:                                                                         entry
+
+# Name: WhileStatement
+# File: statements.js
+# Location: statements.js:42:0
+func WhileStatement():
+0:                                                                         entry
+
```

Comment:
Remove these changes, we don't support these statements on IR. So tests should sill be failing.

Reply:
Wouldn't it be worth leaving it just to avoid confusion for other people working on the engine? I ended up wasting some time trying to understand why my changes broke the tests.

We can leave a TODO that this IR is not valid and must be updated. What do you think?

Reply:
We are expecting that the IR tests fail because we don't support all nodes from ast package on IR. So since the same sample code is used to test the AST parsing and IR parsing, the IR tests should fail. The file `internal/semantic_test.go` contains a basic documentation about these tests.